app-misc/glcdprocdriver/files/glcdprocdriver-0.1.2-makefile.patch

```diff
@@ -0,0 +1,37 @@
+--- a/Make.config	2015-01-03 01:58:18.000000000 +0100
```

Comment:
Why do you need to patch it? It seems you can achieve the same result with a few overrides to `make`.

Reply:
At least not for `CC` and `CXX`?
Yes, for the rest, I could do it, but I thought, patching is better, so the eBuild looks much cleaner without manually calling `emake` etc. Do you still want me to make it pass manually via emake?

Reply:
Patches are good for upstream. On our end, less files is better.

Reply:
Ok, I've stripped down the patch and updated the eBuild.

Reply:
Still, why can't you override `CC` and `CXX`?

Reply:
If `CC = gcc` is definied in Makefile, I don't see any way to override it? But I guess, you know, how this can be done?

Reply:
`emake CC="$(tc-getCC)" ...`

Reply:
Sigh, I see my mistake. I should of course do this in `src_compile`, not `src_install` m( Sorry for that!